backend-stacks/tests/ryder-nft_admin_test.ts

```diff
@@ -70,7 +70,7 @@ Clarinet.test({
         deployer.address
       ),
     ]);
-    block.receipts[0].result.expectErr().expectUint(507);
+    block.receipts[0].result.expectErr().expectUint(507); // err-max-mint-reached
```

Comment:
Should we export all error constants from `ryder-nft-client.ts`?

Reply:
Good idea!